### PR TITLE
Add Minecraft Central presence

### PR DESCRIPTION
**Pull request by Feeeh**
Examples:
image
image
image
image
and much more.

First presence, and kinda first time with js, please don't hate. lul

### CODE REVIEW

File: Minecraft Central/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/W7XIUCd.png",
+    "thumbnail": "https://i.imgur.com/yTWQ5RJ.png",
+    "color": "#da8a00",
+    "tags": [
+        "minecraft",
+        "mccentral"
+      ],
+    "category": "games",
+    "iframe": false
```
Comment (Bas950) [inline Minecraft Central/dist/metadata.json]: can remove the iframe line, dont forget to remove the comma behind games if you do

Comment (Feeeh) [reply Minecraft Central/dist/metadata.json]: Done.

**Comment by doomlerd** [comment]
Hello @Feeeh, your PR is not merged yet because we're making some changes in favor of quality assurance which will require your PR to be re-made, make sure to fix the [current reported issues](https://deepscan.io/dashboard/#view=project&tid=6792&pid=8925&bid=113944&subview=pull-request&prid=300528), and we'll inform you what to do next ( to comply with the new changes ), I'll keep this on hold now until we finish.

For reference : #1248 ( rebase #1258 )

**Comment by doomlerd** [comment]
Sorry for the wait, now that the repo is unlocked you have to recreate the PR with these changes :
format your code with either spaces or tabs and using only 2 or 4 in tab width, then recreate the PR for the new checks to start.

**Pull request closed without merging**